backend/Perflow/Perflow.csproj

```diff
@@ -26,10 +26,10 @@
     <Folder Include="Infrastructure\" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Reference Include="Shared">
-      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
-    </Reference>
+  <ItemGroup>
+    <Reference Include="Shared">
+      <HintPath>..\Shared\Shared\bin\Debug\net5.0\Shared.dll</HintPath>
```

Comment:
Needs to be fixed. Don't add direct references to dll, reference projects instead. Current approach won't work in production